fastlane/Fastfile

```diff
@@ -0,0 +1,127 @@
+##
+## Fastlane configuration for the Nevis Mobile Authentication SDK Android Coroutines Example application.
+## See https://docs.fastlane.tools
+##
+
+require 'java-properties'
+
+default_platform(:android)
+
+root_dir = File.expand_path(File.join(File.dirname(__FILE__), "../"))
+signing_configs_gradle = File.join(root_dir, "app/signing_configs.gradle")
+gradle_properties = File.join(root_dir, "gradle.properties")
+application_module = File.join(root_dir, "app/src/main/java/ch/nevis/exampleapp/coroutines/dagger/ApplicationModule.kt")
+android_manifest = File.join(root_dir, "app/src/main/AndroidManifest.xml")
+build_gradle = File.join(root_dir, "app/build.gradle")
+display_name = "Nevis Mobile Authentication SDK Android Coroutines Example App"
+
+def report_on_success(message)
+    slack(
+        message: "#{message}\nCheck build logs: #{ENV['CURRENT_BUILD_URL']}",
+        slack_url: ENV['SLACK_WEBHOOK_URL']
+    ) if ENV['SLACK_WEBHOOK_URL']
+end
+
+def report_on_error(message, exception)
+    slack(
+        message: "#{message}\nCheck build logs: #{ENV['CURRENT_BUILD_URL']}",
+        slack_url: ENV['SLACK_WEBHOOK_URL'],
+        success: false,
+        attachment_properties: {
+            fields: [
+                {
+                    title: "Error message",
+                    value: exception.to_s,
+                    short: false
+                }
+            ]
+        }
+    ) if ENV['SLACK_WEBHOOK_URL']
+    UI.user_error!("#{exception}")
+end
+
+platform :android do
+    desc "Build and tag the application"
+    desc ""
+    lane :main do |options|
```

Comment:
At least on iOS, we started to document the possible values inside the `options`. I did the same in the case of the Flutter [example app](https://github.com/nevissecurity/nevis-mobile-authentication-sdk-example-app-flutter/blob/179dfc0df953496c68320cfa6ee9d2a9b8f2799e/ios/fastlane/Fastfile#L63).

Reply:
Fixed.